homeassistant/components/zha/binary_sensor.py

```diff
@@ -276,7 +267,7 @@ class AqaraPetFeederErrorDetected(BinarySensor, id_suffix="error_detected"):
 
     SENSOR_ATTR = "error_detected"
     _attr_device_class: BinarySensorDeviceClass = BinarySensorDeviceClass.PROBLEM
-    _attr_name: str = "Error detected"
+    _attr_translation_key: str = "error_detected"
```

Comment:
Do we really need this special translation key? Doesn't Problem already cover it?

Reply:
Yea, let's go with the standard device class name here.